contrib/docker/solr/solrconfig.xml

```diff
@@ -0,0 +1,1462 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
1400 lines of solr config? I guess this is because the new default is now incompatible with ckan?